.github/workflows/unstale.yml

```diff
@@ -0,0 +1,19 @@
+name: Unmark issues and pull requests as stale on activity
+on:
+  issue_comment:
+    types: [created]
+
+# actions/stale does this automatically, but only once a day.
+# This immediately removes the label when the user creates a comment.
+jobs:
+  remove-stale-label:
+    if: (github.repository == 'rubocop/rubocop')
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - name: Remove stale label
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: gh issue edit ${{ github.event.issue.number }} --remove-label "stale" -R ${{ github.repository }}
```

Comment:
Since `if: (github.repository == 'rubocop/rubocop')` is set, my understanding is that this workflow will only run for the `rubocop/rubocop` repository. Given that, is `-R ${{ github.repository }}` still necessary?

Reply:
And `pull-requests: write` is set. With that in mind, wouldn't a similar setting also be necessary for pull requests as shown below?

```diff
+      - name: Remove stale label from pull requests
+        run: gh pr edit ${{ github.event.issue.number }} --remove-label "stale"
```

Reply:
This command makes no distinction between pr and issue, both work fine with just `ìssue`. Similarly, `issue_comment` should also trigger for PRs.

Reply:
`-R` is necessary because it expects a `.git` folder, otherwise it errors (even if it doesn't require it). https://github.com/Earlopain/rubocop/actions/runs/13919661168/job/38949727878